brian2/devices/cpp_standalone/device.py

```diff
@@ -481,6 +481,21 @@ def variableview_get_with_expression(self, variableview, code,
                                   'standalone scripts.')
 
     def code_object_class(self, codeobj_class=None, pref=None):
+        '''
+        Return `CodeObject` class (either `CPPStandaloneCodeObject` class or input)
+
+        Parameters
+        ----------
+        codeobj_class : None or a CodeObject class.
+            If this is keyword is set to None or no arguments are given, this method will return
+            the default (`CPPStandaloneCodeObject` class).
+        fallback_pref : str
+            For the cpp_standalone device this option is ignored.
+
+        Returns
+        -------
+        `CodeObject` class
```

Comment:
The syntax here is a bit wrong, it works like for `Parameters`, i.e. something along the lines of:
```
Returns
-------
codeobj_class : class
    The `CodeObject` class that should be used.
```